### PR TITLE
vcsa collector: charts descritpion and alarms

### PR DESCRIPTION
**Pull request by ilyam8**
##### Summary
This PR adds:
- [X] collector/charts descriptions
- [X] alarms 

From [documentation](https://docs.vmware.com/en/VMware-vSphere/6.5/com.vmware.vsphere.vcsa.doc/GUID-52AF3379-8D78-437F-96EF-25D1A1100BEE.html):

```
 The vCenter Server Appliance API offers health status indicators for several key components of the appliance:
 - green  The component is healthy.
 - yellow The component is healthy, but may have some problems.
 - orange The component is degraded, and may have serious problems.
 - red The component is unavailable, or will stop functioning soon.
 - gray No health data is available.
```

Alarms added for following components:
  - applmgmt
  - databasestorage
  - load
  - mem
  - softwarepackages
  - storage
  - swap
  - system

##### Component Name
[/web/gui](https://github.com/netdata/netdata/tree/master/web/gui)
[health/health.d](https://github.com/netdata/netdata/tree/master/health/health.d)

##### Additional Information

Ref: https://github.com/netdata/go.d.plugin/pull/260

### CODE REVIEW

**Comment by ilyam8** [comment]
i tested the PR:
 - install from source `[OK]`
 - info on the dashboard `[OK]`
 - all alarms are up `[OK]`

File: health/health.d/vcsa.conf [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+   every: 10s
+    warn: $this == 1
+    crit: ($this == 2) || ($this == 3)
+   delay: down 1m multiplier 1.5 max 1h
+    info: appl mgmt health status
+      to: sysadmin
+
+
+# Software updates health:
+#  - 0: no updates available.
```
Comment (thiagoftsm) [inline health/health.d/vcsa.conf]: Hi @ilyam8 ,

Overall the alarms are fine, but I did not approve yet due a doubt here, I saw that in this group of comments we are missing the status "- 1", is there a special motive for this?

Comment (ilyam8) [reply health/health.d/vcsa.conf]: There is no `-1` for software updates according documentation

see https://vmware.github.io/vsphere-automation-sdk-rest/vsphere/index.html#SVC_com.vmware.appliance.health

> Get information on available software updates available in the remote vSphere Update Manager repository.
> Red indicates that security updates are available.
> Orange indicates that non-security updates are available.
> Green indicates that there are no updates available.
> Gray indicates that there was an error retreiving information on software updates.

^^ no yellow (-1) status as you see

Comment (ilyam8) [reply health/health.d/vcsa.conf]: Hawk eye btw 😄

Comment (thiagoftsm) [reply health/health.d/vcsa.conf]: Thank you by the explanation :smile:

Comment (ilyam8) [reply health/health.d/vcsa.conf]: Just noticed, it is not `-1` it is `1`, i added `-` to show this is a list.

**Pull request merged**